performance-evaluation/google_drive.py

```diff
@@ -0,0 +1,53 @@
+from constants import GDRIVE_FILE_NAME
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+from percol import Percol
+import readchar
+
+
+def select_from_list(folders):
+    si, so, se = "", "", ""
+    candd = [p[0] for p in folders]
+    with Percol(
+            actions=[],
+            descriptors={'stdin': si, 'stdout': so, 'stderr': se},
+            candidates=iter(candd)) as p:
+        p.loop()
+    results = p.model_candidate.get_selected_result()
+    ret = [pp for pp in folders if pp[0] == results]
```

Comment:
First, you need a better name for `pp` and `ret` lol, and I've seen that later on you use `results[0][0]` and `results[0][1]` so I guess it should be better if this function simply returns `[0]` for the `ret` or whatever you call that variable now, instead of returning the full list since the user has no control over what option would be chosen from that list.

Reply:
Done